tmt/export.py

```diff
@@ -46,21 +46,22 @@ def export_to_nitrate(test, create, general):
     try:
         nitrate_id = test.node.get('extra-nitrate')[3:]
         nitrate_case = nitrate.TestCase(int(nitrate_id))
-        nitrate_case.summary # Make sure we connect to the server now
+        nitrate_case.summary  # Make sure we connect to the server now
         echo(style(f"Test case '{nitrate_case.identifier}' found.", fg='blue'))
     except TypeError:
         # Create a new nitrate test case
         if create:
             nitrate_case = create_nitrate_case(test)
             new_test_created = True
+            test._metadata['extra-summary'] = nitrate_case.summary
         else:
             raise ConvertError("Nitrate test case id not found.")
     except (nitrate.NitrateError, gssapi.raw.misc.GSSError) as error:
         raise ConvertError(error)
 
     # Summary
-    summary = test.node.get(
-        'extra-summary', test.node.get('extra-task', test.summary))
+    summary = test._metadata.get('extra-summary') \
+        or test._metadata.get('extra-task') or test.summary or ''
```

Comment:
Should we update summary for existing cases? IMHO yes so 'test.summary' should be transformed into `repo/name - summary`

Reply:
I think this should only apply for new tests as I don't believe people would like `tmt` changing their tests out of the blue.
Can you expand on why you think it should change existing tests?

Reply:
Because of 'sync' feature of tmt export --nitrate. But good point. this is something to first agree on and get wider confirmation. --> out of scope of this issue.